dbt/adapters/presto/connections.py

```diff
@@ -154,13 +156,17 @@ def open(cls, connection):
                 credentials.user,
                 credentials.password,
             )
+            if credentials.http_scheme and credentials.http_scheme != "https":
+                raise dbt.exceptions.RuntimeException("http_scheme must be set to 'https' for 'ldap' method.")
             http_scheme = "https"
         elif credentials.method == 'kerberos':
             auth = prestodb.auth.KerberosAuthentication()
+            if credentials.http_scheme and credentials.http_scheme != "https":
+                raise dbt.exceptions.RuntimeException("http_scheme must be set to 'https' for 'kerberos' method.")
```

Comment:
Some duplication between the handling of `credentials.http_scheme` between the `kerberos` and `ldap` here but I'm hesitant to refactor it because I believe it's coincidental duplication. What do you think @jtcohen6 ?

Reply:
Agreed, the duplication here feels quite minor, and I appreciate having the check be explicit in both places